denys_filichkin/task_5/task_pwsh.sh

```
@@ -0,0 +1,40 @@
+#!/usr/bin/env pwsh
+echo "Hello task 5"
+
+# test 1
+param (
```

Shellcheck (0.10.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)